### PR TITLE
Makes cargo be able to buy more sec gear!

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
[Changelogs]
Adds 3 crates!
2 Dragnets for 3500
2 Tatical tasers for 8000
3 Mini Eguns for 5000
[why]
Drag net guns are good anit-riot and useful if known how to use.
Mini-Eguns are sometimes sec guns, small and likely can be holdout for Heads of Staff or sec or something.
Tatical tasers are legit just lesser AEG that only have a few shots.

### CODE REVIEW

File: code/modules/cargo/packs.dm [hunk 1]
```diff
@@ -620,6 +638,13 @@
 					/obj/item/clothing/gloves/combat)
 	crate_name = "swat crate"
 
+/datum/supply_pack/security/armory/swattasers //Lesser AEG tbh
+	name = "SWAT tatical tasers Crate"
+	desc = "Contains two tactical energy gun, these guns are able to tase, dislabe and lethal as well as hold a seclight. Requires Armory access to open."
```
Comment (Poojawa) [inline code/modules/cargo/packs.dm]: dislabe

File: code/modules/cargo/packs.dm [hunk 2]
```diff
@@ -559,6 +567,16 @@
...
 
+/datum/supply_pack/security/armory/miniguns
+	name = "Personal Miniature Energy Guns"
+	desc = "Contains three miniature energy guns. Each gun has a dislaber and a lethal option. Requires Armory access to open."
+	cost = 5000
+	contains = list(/obj/item/gun/energy/e_gun/mini,
+					/obj/item/gun/energy/e_gun/mini,
+					/obj/item/gun/energy/e_gun/mini)
+	crate_name = "personal engery guns crate"
+	crate_type = /obj/structure/closet/crate/secure/plasma
```
Comment (Poojawa) [inline code/modules/cargo/packs.dm:578]: shouldn't this be a security crate

Comment (Trilbyspaceclone) [reply code/modules/cargo/packs.dm:578]: Other Eguns are all in plasma crates lore wise do to being plasma based weaponds

File: code/modules/cargo/packs.dm [hunk 3]
```diff
@@ -512,6 +512,14 @@
 					/obj/item/storage/belt/bandolier)
 	crate_name = "combat shotguns crate"
 
+/datum/supply_pack/security/armory/dragnetgun
+	name = "DRAGnet gun Crate"
+	desc = "Contains two DRAGnet gun. A Dynamic Rapid-Apprehension of the Guilty net the revolution in law enforcement technology that YOU Want! Requires Armory access to open."
+	cost = 3500
+	contains = list(/obj/item/gun/energy/e_gun/dragnet,
+					/obj/item/gun/energy/e_gun/dragnet)
+	crate_name = "anit riot net guns crate"
```
Comment (silicons) [inline code/modules/cargo/packs.dm]: anit

File: code/modules/cargo/packs.dm [hunk 4]
```diff
@@ -559,6 +567,16 @@
...
 	dangerous = TRUE
 
+/datum/supply_pack/security/armory/miniguns
+	name = "Personal Miniature Energy Guns"
+	desc = "Contains three miniature energy guns. Each gun has a dislaber and a lethal option. Requires Armory access to open."
+	cost = 5000
+	contains = list(/obj/item/gun/energy/e_gun/mini,
+					/obj/item/gun/energy/e_gun/mini,
+					/obj/item/gun/energy/e_gun/mini)
+	crate_name = "personal engery guns crate"
```
Comment (silicons) [inline code/modules/cargo/packs.dm]: energy

File: code/modules/cargo/packs.dm [hunk 5]
```diff
@@ -559,6 +567,16 @@
 	crate_type = /obj/structure/closet/crate/secure/plasma
 	dangerous = TRUE
 
+/datum/supply_pack/security/armory/miniguns
+	name = "Personal Miniature Energy Guns"
+	desc = "Contains three miniature energy guns. Each gun has a dislaber and a lethal option. Requires Armory access to open."
```
Comment (silicons) [inline code/modules/cargo/packs.dm]: disabler

**Review by silicons — APPROVED** [review]

**Pull request merged**